migrations/1580422185800_alter_table_public_recipe_drop_column_variation_count/down.yaml

```diff
@@ -0,0 +1,11 @@
+- args:
```

Comment:
I am guessing you made these db changes through the hasura console GUI.... did hasura make these table edit queries or did you have to produce these?

Reply:
I didn't have to make these! I just clicked the same buttons in the online gui and they get autogenerated in this folder here 🙂 pretty slick!